extensions/perplexity/src/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Action, ActionPanel, Form, open, popToRoot } from "@raycast/api";
+
+type Values = {
+  query: string;
+  copilot: boolean;
+  focus: string;
+};
+
+export default function Command() {
+  function handleSubmit({ query: q, copilot, focus }: Values) {
+    const params = new URLSearchParams({
```

Comment:
You might want to validate that query is not empty and show a failure tost here

Reply:
Ah, good call! Went with an error message instead to put the feedback closer to the input.